README.md

```diff
@@ -7,14 +7,14 @@ It is a work in progress at this moment. You will find the individual fiscalizat
 
 | **Project** | **Windows** | **Linux** | **Notes** |
 | ----------- | ----------- | --------- | --------- |
-| All | [![Build](https://img.shields.io/github/workflow/status/MewsSystems/fiscalizations/Build%20and%20test%20-%20All%20(Windows)/master)](https://github.com/MewsSystems/fiscalizations/actions/workflows/build-and-test-all-windows.yml) | [![Build](https://img.shields.io/github/workflow/status/MewsSystems/fiscalizations/Build%20and%20test%20-%20All%20(Linux)/master)](https://github.com/MewsSystems/fiscalizations/actions/workflows/build-and-test-all-linux.yml) |
+| All | [![Build](https://img.shields.io/github/workflow/status/MewsSystems/fiscalizations/Build%20and%20test%20-%20All%20(Windows)/master)](https://github.com/MewsSystems/fiscalizations/actions/workflows/build-and-test-all-windows.yml) | [![Build](https://img.shields.io/github/workflow/status/MewsSystems/fiscalizations/Build%20and%20test%20-%20All%20(Linux)/master)](https://github.com/MewsSystems/fiscalizations/actions/workflows/build-and-test-all-linux.yml) | Runs on ubuntu-18.04 because our Spanish test certificate is not secure enough to run on Ubuntu-latest. |
```

Comment:
```suggestion
| All | [![Build](https://img.shields.io/github/workflow/status/MewsSystems/fiscalizations/Build%20and%20test%20-%20All%20(Windows)/master)](https://github.com/MewsSystems/fiscalizations/actions/workflows/build-and-test-all-windows.yml) | [![Build](https://img.shields.io/github/workflow/status/MewsSystems/fiscalizations/Build%20and%20test%20-%20All%20(Linux)/master)](https://github.com/MewsSystems/fiscalizations/actions/workflows/build-and-test-all-linux.yml) | Runs on ubuntu-18.04 because our Spanish test certificate is not secure enough to run on Ubuntu-latest. Hungarian tests were disabled. |
```